CHANGES.rst

```diff
@@ -424,8 +430,16 @@ astropy.wcs
 Other Changes and Additions
 ---------------------------
 
+- Minimum version of supported Python is now 3.7. [#10900]
+
 - Minimum version of supported Numpy is now 1.17. [#10664]
 
+- Minimum version of supported Scipy is now 1.1. [#10900]
+
+- Minimum version of supported PyYAML is now 3.13. [#10900]
+
+- Minimum version of supported Matplotlib is now 3.0. [#10900]
+
```

Comment:
Nitpicking on verbs, but I think required is more precise over supported. In the past, we used both, though with the supported it was the negate, e.g. saying that older than these are not supported.

Reply:
Yeah, that makes sense. I changed it as requested. Thanks!